src/ImageCard/ImageCard.js

```diff
@@ -1,8 +1,32 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import './ImageCard.scss'
 
 const ImageCard = () => {
+  const [spans, setSpans] = useState(0);
 
+  const imageRef = React.createRef();
 
-  return(<h1>ImageCard</h1>)
-}
\ No newline at end of file
+  useEffect(() => {
+    imageRef.current.addEventListener('load', setImageSpans);
+  }, [])
+  
+  const setImageSpans = () => {
+    const height = imageRef.current.clientHeight;
+    const cardSpans = Math.ceil(height / 20);
+
+    setSpans(cardSpans);
+  };
+  // const { description, urls } = props.image;
+
+  return (
+    <div style={{gridRowEnd: `span ${spans}`}}>
+      <img 
+        ref={imageRef} 
```

Comment:
Hard coded values to remove errors in testing, just need to hide api key and connect api to app